drivers/sensor/bmi160/bmi160.c

```diff
@@ -709,6 +724,128 @@ static int bmi160_attr_set(const struct device *dev, enum sensor_channel chan,
 	return 0;
 }
 
+static int bmi160_attr_get(const struct device *dev, enum sensor_channel chan,
```

Comment:
New feature isn't related to the commit message `bmi160: Make changes to align driver to datasheet`